practiceCourseGenerator/build.sh

```diff
@@ -0,0 +1,106 @@
+#!/bin/bash
+
+#build.sh
+
+# Build script to assemble PracticeCourseGenerator.
+# Creates a tar containing version.txt and practiceCourseGenerator.py
+
+set -x
+
+# return a sortable timestamp as a string without a newline on the end.
+function niceTimestamp {
+    echo $(date +"%F-%H-%M")
+}
+
+function atStep {
+    local msg=$1
+    echo "+++ $1"
+}
+
+# make a clean directory to hold any build ARTIFACTS
+function makeARTIFACTSDir {
+
+    if [ -e ./ARTIFACTS ]; then
+        rm -rf ./ARTIFACTS;
+    fi
+
+    mkdir ./ARTIFACTS
+}
+
+# make a clean directory to hold any build ARTIFACTS
```

Comment:
Or rather, don't make the same comment twice.  Misleading comments are worse than no comments at all.